src/components/Search/algolia/index.js

```diff
@@ -15,6 +15,7 @@ import styled from '@emotion/styled';
 import SearchStatus from '../Status';
 import Pagination from './pagination';
 import Stats from './stats';
+import AlgoliaLogo from './AlgoliaLogo';
```

Comment:
So to use a free plan/open source project, we need to put their logo in our website right?

Reply:
Yes